packages/ppxlib/ppxlib.0.18.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/ocaml-ppx/ppxlib"
+bug-reports: "https://github.com/ocaml-ppx/ppxlib/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ppxlib.git"
+doc: "https://ocaml-ppx.github.io/ppxlib/"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs] { ocaml:version >= "4.10" }
+]
+depends: [
+  "ocaml"                   {>= "4.04.1"}
```

Comment:
does it support OCaml 4.12?

`ppxlib.0.17.0` fails with:
```
#=== ERROR while compiling ppxlib.0.17.0 ======================================#
# context              2.1.0~beta2 | linux/x86_64 | ocaml-variants.4.12.0+trunk | file:///home/opam/opam-repository
# path                 ~/.opam/4.12.0+trunk/.opam-switch/build/ppxlib.0.17.0
# command              ~/.opam/4.12.0+trunk/bin/dune build -p ppxlib -j 71
# exit-code            1
# env-file             ~/.opam/log/ppxlib-6-af3201.env
# output-file          ~/.opam/log/ppxlib-6-af3201.out
### output ###
# [...]
#           pp ast/location_error.pp.mli (exit 1)
# (cd _build/default && ast/pp/pp.exe 4.12.0+dev0-2020-04-22 ast/location_error.mli) > _build/default/ast/location_error.pp.mli
# Unknown OCaml version 4.12.0+dev0-2020-04-22
#           pp ast/versions.pp.mli (exit 1)
# (cd _build/default && ast/pp/pp.exe 4.12.0+dev0-2020-04-22 ast/versions.mli) > _build/default/ast/versions.pp.mli
# Unknown OCaml version 4.12.0+dev0-2020-04-22
#           pp ast/location_error.pp.ml (exit 1)
# (cd _build/default && ast/pp/pp.exe 4.12.0+dev0-2020-04-22 ast/location_error.ml) > _build/default/ast/location_error.pp.ml
# Unknown OCaml version 4.12.0+dev0-2020-04-22
#           pp ast/versions.pp.ml (exit 1)
# (cd _build/default && ast/pp/pp.exe 4.12.0+dev0-2020-04-22 ast/versions.ml) > _build/default/ast/versions.pp.ml
# Unknown OCaml version 4.12.0+dev0-2020-04-22
```
(tested with https://github.com/ocaml-ppx/ocaml-migrate-parsetree/pull/107)

Reply:
I got an answer to my question by looking into it myself. I opened a PR to fix this issue upstream: https://github.com/ocaml-ppx/ppxlib/pull/190